Containerfile

```diff
@@ -5,12 +5,14 @@ ARG base_image=quay.io/app-sre/centos:8
 ARG package_manager
 FROM $base_image
 
+# GIT_EDITOR: git cherry-pick --continue prompts editor, this is the only way to make it non-interactive
```

Comment:
Hmmm, that's what I wanted to avoid originally, but let's go with this and try to improve later. I really understand it's really hard -- there are multiple modes, path specified x not-specified and force and other git options.

Reply:
the only reason I am doing this is that `git cherry-pick --continue` prompts an editor and there is no way to suppress it

Reply:
Yes, I know that... It's same with interactive rebase that can't be used without this trick.